knpu/episode3/abstract-ship.md

```diff
@@ -36,48 +49,89 @@ This is a way of changing the class heirachy so that each class has only what it
 
 ## Creating an AbstractShip
 
-Let's start this! Create a new PHP Class called `AbstractShip`, since it is the most abstract idea of
-a ship in our project. To start, I'm going to copy everything out of the `Ship` class and paste it
-into `AbstractShip`. I know this looks like where we just were, but trust me we're going somewhere with
-this.
+Let's start this! Create a new PHP Class called `AbstractShip`:
+
+[[[ code('7e9d688d11') ]]]
+
+Since it is the most abstract idea of a ship in our project. To start, I'm going to copy everything
+out of the `Ship` class and paste it into `AbstractShip`:
+
+[[[ code('3d459c1edc') ]]]
+
+I know this looks like where we just were, but trust me we're going somewhere with this.
+
+Now, let's write `Ship extends AbstractShip`:
+
+[[[ code('90643e4c44') ]]]
+
+and do the same thing in `RebelShip` changing it from `Ship` to `AbstractShip`:
```

Comment:
I usually try to always capitalize the first letter - unless this line is definitely not a full sentence, for example:

```md
Update Ship

<code>

and JediShip

<code>
```